studioShared/studioConfig.tsx

```diff
@@ -12,8 +18,32 @@ const config: WorkspaceOptions = {
   basePath: "/shared",
   projectId,
   dataset,
-  schema,
-  plugins: [structureTool()],
+  schema: {
+    ...schema,
+    templates: (prev) =>
+      prev.filter((template) => template.value.language === i18n.base),
+  },
+  plugins: [
+    structureTool({
+      structure: deskStructure,
+    }),
+    visionTool({ defaultApiVersion: apiVersion }),
+    documentInternationalization({
+      // TODO: a function that takes the client and returns a promise of an array of supported languages
+      // MUST return an "id" and "title" as strings
+      // supportedLanguages: (client) => client.fetch(`*[_type == "language"]{id, title}`),
+      supportedLanguages: i18n.languages,
+      schemaTypes: [customerCaseID],
+      languageField: `language`,
+      metadataFields: [defineField({ name: "slug", type: "slug" })],
```

Comment:
Maybe import `titleSlug` instead of re-defining the field?

Reply:
removing as it's not needed and just came with my careless copy paste! 😅